NettyMessagingServiceTest.shouldCompleteExistingRequestFutureExceptionallyWhenMessagingServiceIsClosed flaky

**Summary**

- How often does the test fail? at least once so far
- Does it block your work? no
- Do we suspect that it is a real failure? unknown

**Failures**

<details><summary>Example assertion failure</summary>
<pre>
java.lang.AssertionError: 

Expecting
  <CompletableFuture[Incomplete]>
to be completed exceptionally.
Be aware that the state of the future in this message might not reflect the one at the time when the assertion was performed as it is evaluated later on
	at io.atomix.cluster.messaging.impl.NettyMessagingServiceTest.shouldCompleteExistingRequestFutureExceptionallyWhenMessagingServiceIsClosed(NettyMessagingServiceTest.java:213)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeLazy(JUnitCoreWrapper.java:119)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:87)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:157)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
</pre>
</details>

**Hypotheses**


**Logs**

<details><summary>Logs</summary>
<pre>
<![CDATA[02:42:15.009 [] INFO  io.zeebe.test - Test finished: shouldCompleteFutureExceptionallyIfMessagingServiceIsClosedInBetween(io.atomix.cluster.messaging.impl.NettyMessagingServiceTest)
02:42:15.009 [] INFO  io.zeebe.test - Test started: shouldCompleteExistingRequestFutureExceptionallyWhenMessagingServiceIsClosed(io.atomix.cluster.messaging.impl.NettyMessagingServiceTest)
02:42:15.095 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:44615
02:42:15.096 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
02:42:15.103 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:34089
02:42:15.103 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
02:42:15.111 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:32899
02:42:15.112 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
02:42:15.120 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:41287
02:42:15.120 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
02:42:15.199 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:39497
02:42:15.200 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
02:42:15.208 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:44625
02:42:15.208 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
02:42:15.211 [] DEBUG io.atomix.cluster.messaging.impl.NettyMessagingService - Writing client protocol version V2 for connection to zeebe-ci-build-develop-1495-ff6nr-nq4hs/10.100.14.9:34089
02:42:15.298 [] DEBUG io.atomix.cluster.messaging.impl.NettyMessagingService - Activating server protocol version V2 for connection to /10.100.14.9:38334
02:42:17.405 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Stopped
02:42:19.618 [] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Stopped
</pre>
</details>

--- a/atomix/cluster/src/test/java/io/atomix/cluster/messaging/impl/NettyMessagingServiceTest.java
+++ b/atomix/cluster/src/test/java/io/atomix/cluster/messaging/impl/NettyMessagingServiceTest.java
@@ -204,7 +204,7 @@ public class NettyMessagingServiceTest {
     final String subject = nextSubject();
     final CompletableFuture<byte[]> response =
         netty1.sendAndReceive(
-            address2, subject, "hello world".getBytes(), false, Duration.ofSeconds(5));
+            address2, subject, "hello world".getBytes(), true, Duration.ofSeconds(5));
 
     // when
     netty1.stop().join();

